GVFS/GVFS.UnitTests/Git/GitAuthenticationTests.cs

```diff
@@ -78,38 +81,15 @@ public void ContinuesToBackoffIfTryGetCredentialsFails()
             dut.IsBackingOff.ShouldEqual(true, "Should continue to backoff if failed to get credentials");
         }
 
-        [TestCase]
-        public void GitProcessFailuresAreRetried()

```

Comment:
It was not a very useful test to begin with, and is no longer applicable at all.

This test was trying to confirm that if we fail to auth because of a failure to call the git process (not an actual auth failure), we don't back off and retry right away. However, the product code doesn't actually have that behavior. This test happened to work but only if no call to `git credential` had ever succeeded. Once the first one passes, any subsequent failures cause a backoff.

Two thoughts for removing the test. 1) It's very uncommon that `git credential` fails for any reason other than an actual auth failure and 2) the condition (that this is the first ever call to `git credential` that failed) can no longer even happen, because now all callers have to call `TryInitialize`, which will call `git credential` if needed. And so now, any call to `TryGetCredentials` that fails is guaranteed to not be the first one.

If we ever want to not back off in this scenario, we'll have to actually add in that feature.